va/compat_win32.h

```diff
@@ -46,6 +46,13 @@ typedef LONG NTSTATUS;
 
 typedef unsigned int __uid_t;
 
+#define getenv _getenv
```

Comment:
Please explain in the commit message what the issue is. Referring to a test (which isn't yet merged) is not useful.

Also you might want to a) include the `#ifdef` guards here (as opposed to fixup in next commit) and b) split it into separate PR - since this is seemingly a fix independent of the file/path shenanigans.

Reply:
I'll add more info to the commit and do the fixes. Looks like it can also be a separate PR that goes in before this one. The tests do exist, the draft just adds more of the same test class (vaInitialize_vaTerminate). I'll split out the commits in a second.

Reply:
compat_win32: Add getenv wrapper for WIN32 #687